### DISCUSSION TITLE
uniapp use font

### DISCUSSION THREAD

**Discussion by GuoSirius** [FortAwesome/Font-Awesome#20905] [Feature Requests]
### Feature description

uniapp use font

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Could you please edit the first post and clarify this request?
